src/page/product-list.page.ts

```diff
@@ -1,14 +1,22 @@
-import { $, ElementFinder, browser, ExpectedConditions } from 'protractor';
+import { $$, browser, ExpectedConditions, ElementArrayFinder } from 'protractor';
 
 export class ProductListPage {
-  private addToCartBtn: ElementFinder;
+  private products: ElementArrayFinder;
 
   constructor () {
-    this.addToCartBtn = $('a.button.ajax_add_to_cart_button.btn.btn-default');
+    this.products = $$('.product-container');
```

Comment:
what does the `$$`? is that a short cut for something?

Reply:
Yes! It works like the function element.all of protractor. It is used to specify that i need the array of elements within a specific parent. That's why the class attribute products is an ElementArrayFinder. If I use only $ like before is because I only need one specific element.